.github/workflows/cypress-ete-okta.yml

```diff
@@ -0,0 +1,61 @@
+name: cypress-ete-okta
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: ['review_requested']
+jobs:
+  cypress-ete-okta:
+    name: Cypress (ete using okta)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version-file: '.nvmrc'
+          cache: 'yarn'
+      - name: Cypress run
+        uses: cypress-io/github-action@v2
+        env:
+          PORT: 8861
+          ENVIRONMENT: production
```

Comment:
Is this meant to say production? Does this variable even get used anymore?

Reply:
hm, I think the only place it's used is to set the `mode` in our webpack configuration. can probably be removed in a separate pr in favour of checking the `STAGE` variable  (as it'll need to be wiped from our .env files in aws etc)